docs/one-wire-devices.md

```diff
@@ -0,0 +1,150 @@
+---
+title: One-Wire Devices
+---
+
+One-wire devices including but not limited to temperature loggers, battery monitors, humidity sensors, voltage and current sensors can be used with a wide range of controllers and processors in the Omega2 family like Omega2, Omega2+, Omega2S, and more through One-wire communication protocol. 
```

Comment:
Can we simplify this to just say used with "the Omega2 family of modules"
Note that the Omega2/2+/2S/2S+ are variants of the same product